web/src/App.vue

```diff
@@ -17,111 +17,7 @@ export default defineComponent({
 
 <template>
   <v-app>
-    <v-app-bar
-      app
-      color="white"
-      clipped-left
-      elevation="1"
-    >
-      <a
-        style="height: 100%"
-        @click="setConditions([])"
-      >
-        <img
-          src="/NMDC_logo_long.jpg"
-          height="100%"
-          alt="Page home"
-        >
-      </a>
-      <v-spacer />
-
-      <v-btn
-        depressed
-        small
-        href="https://nmdc-documentation.readthedocs.io/en/latest/tutorials/nav_data_portal.html"
-        target="_blank"
-        rel="noopener noreferrer"
-        class="mx-2"
-      >
```

Comment:
Is there going to be a place for these data portal tutorial/documentation links? I'm not seeing them anywhere in the new app bar.

Reply:
Yeah that's kind of what I meant when I was alluding to iterating on the content of the menus. But actually let's hold off on merging this for right now and I'll see if I can get some engagement team members to provide some input first.

Reply:
See latest comment